### DISCUSSION TITLE
TuYa Smart Plug TS011F Live-to-Neutral issue

### DISCUSSION THREAD

**Discussion by fsevilla3** [Koenkk/zigbee2mqtt#13765] [Devices]
This isn’t really about Z2M but I thought I’d ask here. I just got some [generic TuYa smart plugs with power monitoring](https://a.aliexpress.com/_m0ZOhBa), which Z2M recognises as [TS011F_plug_1](https://www.zigbee2mqtt.io/devices/TS011F_plug_1.html). I tested them using a socket tester and found that when they’re switched off, there’s voltage (approx 240V) detected between live and neutral. Normally there shouldn’t be any voltage detected when the plug is off, and I’ve tested this with other smart plug brands. Has anyone using a TuYa TS011F tested their plug? What results are you getting? I’m trying to gauge if this is an issue with the seller or if all of the generic ones that look like this will have the same issue. Thanks!

**Comment by TheQue42** [comment]
I just tested one of mine, and it seems the socket only switches/disconnects ONE of the outputs of the device. So depending on the orientation when I push it into the wall socket, it might disconnected the neutral or the phase "hole", once I press the button to "off" the socket.
Bad of course, and I am scolding myself for not doublechecking this myself, before I bought 10 of them... But, I suppose you get what you pay for :-/
